### PR TITLE
chore: Bump version to 3.0.0-beta.1

### PR DESCRIPTION
**Pull request by riddell-stan**
Bump version. The PR also has a commit which removes the need to update __version__ manually.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -50,3 +46,7 @@ include_trailing_comma = true
 force_grid_wrap = 0
 use_parentheses = true
 line_length = 119
+
+[build-system]
```
Comment (riddell-stan) [inline pyproject.toml:50]: `poetry` moved this here when I used `poetry version`. I gather it belongs last.

File: stan/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 from stan.model import build  # noqa
 
-__version__ = "3.0.0-beta.0"
+__version__ = "3.0.0-beta.1"
```
Comment (ahartikainen) [inline stan/__init__.py:3]: We still need to update this manually?

Comment (riddell-stan) [reply stan/__init__.py:3]: For this version.

**Comment by riddell-stan** [comment]
Merged without review because this is trivial.

**Pull request merged**